src/pages/index.tsx

```diff
@@ -1,29 +1,11 @@
+import AttractionsPage from "@components/AttractionsPage";
 import withAuth from "@utils/withAuth";
-import { useRouter } from "next/router";
-import React from "react";
-import { removeCookie } from "typescript-cookie";
-import { Button } from "../components/Button";
-import PageWrapper from "../components/PageWrapper";
-import { UserContext } from "../contexts/userContext";
+import { GetStaticProps } from "next";
 
-const Dashboard = () => {
-	const router = useRouter();
-	const { userObject, clearUser } = React.useContext(UserContext);
-	const logout = () => {
-		removeCookie("accessToken");
-		router.push("login").then(() => {
-			clearUser();
-		});
-	};
-	return (
-		<div className="container mx-auto my-10 max-w-800px">
-			<PageWrapper>
-				DASHBOARD
-				<h2>You are logged in{userObject ? ` as ${userObject.firstName} ${userObject.lastName}` : ""}</h2>
-				<Button onClick={logout} label="Logout" />
-			</PageWrapper>
-		</div>
-	);
-};
+export const getStaticProps: GetStaticProps = async (context) => ({
+	props: {
+		messages: (await import(`../../i18n/${context.locale}.json`)).default,
+	},
+});
 
-export default withAuth(Dashboard);
+export default withAuth(AttractionsPage);
```

Comment:
I started externalizing some pages into the `/components` directory, where I would like to collect all page components at some point! When I saw how many pages we already have, I stopped with that for now and would continue with this refactoring on demand when tackling each page individually. ✌️